src/ansys/fluent/core/solver/flobject.py

```diff
@@ -870,6 +870,8 @@ def get_cls(name, info, parent=None):
         user_creatable = info.get("user-creatable?", False) or info.get(
             "user_creatable", False
         )
+        if version == "222":
+            user_creatable = True
```

Comment:
`user-creatable?` attribute doesn't exist in 222 settings API, so the `True` is the correct default for 22.2.

Reply:
whereas `False` is the 23.1 default, as `user-creatable? = #t` is only marked where it is true.

Reply:
We should update the Fluent version of the file even if the new code is redundant there.